packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -833,3 +850,15 @@ void _validateUrlString(String url) {
     throw ArgumentError(e);
   }
 }
+
+/// Simple credentials wrapper for authorization requests
+class AuthInfo {
```

Comment:
As a top-level public class, this should probably have a less collision-prone name, like `WebViewAuthInfo`.